apps/web/src/pages/Profile/ExperienceAndSkillsPage/ExperienceAndSkillsPage.stories.tsx

```diff
@@ -56,3 +67,7 @@ WithExperiencesMissingSkills.args = {
     optionalSkills: mockOptionalSkills,
   },
 };
+
+AddExperienceDialogOpen.args = {
+  defaultOpen: true,
+};
```

Comment:
adding the `OverlayOrDialogDecorator` to the parameters here could help with the chromatic snapshot 👍

Reply:
It worked!
[Add dialog decorator](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5701/commits/747f6cec35077cafb334eb3954f2167a9907ec07)
The dialog is too tall to be fully snapshotted but that's a different problem.  :laughing:

Reply:
Awesome!